pkg/resource/resource_types.go

```diff
@@ -140,13 +140,15 @@ var supportedTypes = map[string]ResourceTypeMeta{
 	"aws_appautoscaling_policy":           {},
 	"aws_appautoscaling_scheduled_action": {},
 	"aws_apigatewayv2_api": {children: []ResourceType{
-		"aws_apigatewayv2_route",
+		"aws_apigatewayv2_deployment",
```

Comment:
We don't need this one simply because we don't depend on an `aws_apigatewayv2_api` to directly create a `aws_apigatewayv2_deployment` (e.g. no middleware that will loop over apis to find corresponding deployment like we did for the route our integration).